src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -751,10 +751,8 @@ void processZoneEquipmentInput(EnergyPlusData &state,
                                Array1D_int &NodeNums)
 {
     static constexpr std::string_view RoutineName("processZoneEquipmentInput: "); // include trailing blank space
-    std::string_view zsString = "Zone";
     int spaceFieldShift = 0;
     if (isSpace) {
-        zsString = "Space";
         spaceFieldShift = -1;
     }
```

Comment:
[src/EnergyPlus/DataZoneEquipment.cc:754]:(style),[variableScope],The scope of the variable 'zsString' can be reduced.
[src/EnergyPlus/DataZoneEquipment.cc:754]:(style),[unreadVariable],Variable 'zsString' is assigned a value that is never used.
[src/EnergyPlus/DataZoneEquipment.cc:757]:(style),[unreadVariable],Variable 'zsString' is assigned a value that is never used.